shared/test/loop_mocha_utils.js

```diff
@@ -270,7 +270,7 @@ var LoopMochaUtils = (function(global, _) {
     var consoleWarn = console.warn;
     var consoleError = console.error;
     console.warn = function() {
-      var args = Array.slice(arguments);
+      var args = Array.prototype.slice.call(arguments);
```

Comment:
These changes were to fix the failure reporting in Karma when it was running the tests in Google Chrome - Chrome requires the more explicit form.